games/f.yaml

```diff
@@ -225,7 +225,9 @@
 
 - name: [Football Manager, Football Manager (1982 series)]
   meta:
-    genre: [simulation]
```

Comment:
Why isn't this a simulation game? Also I disagree again with the bland "strategy" genre. Games are rarely categorized as "strategy" and either TBS or RTS is always used as the case instead.